### PR TITLE
Update new/edit bookmark

### PR DESCRIPTION
**Pull request by sambuddhabasu**

### CODE REVIEW

**Comment by craigmaloney** [comment]
Looks good to me. :+1:

**Comment by preddy5** [comment]
:+1:

File: bookie/tests/test_webviews/test_webviews.py [hunk 1]
```diff
@@ -140,6 +143,106 @@ def test_existing_url_entry_error(self):
...
+
+        # Add the private bookmark.
+        res = self.app.post(
+            '/admin/new_error',
+            params={
+                'url': test_url_private,
+                'description': '',
+                'extended': '',
+                'tags': '',
+                'is_private': '',
```
Comment (mitechie) [inline bookie/tests/test_webviews/test_webviews.py]: I think what gets posted is is_private: 'on'. Please double check and make sure it matches what it actually sent by the browser.

File: bookie/tests/test_webviews/test_webviews.py [hunk 2]
```diff
@@ -140,6 +143,106 @@ def test_existing_url_entry_error(self):
...
+
+        # Make the bookmark private.
+        res = self.app.post(
+            '/admin/edit_error/{0}'.format(bmark.hash_id),
+            params={
+                'url': test_url,
+                'description': '',
+                'extended': '',
+                'tags': '',
+                'is_private': '',
```
Comment (mitechie) [inline bookie/tests/test_webviews/test_webviews.py]: same here

**Comment by mitechie** [comment]
:+1: with the one extra check. Thanks for this update.

File: bookie/tests/test_webviews/test_webviews.py (hunk 1 above)
Comment (sambuddhabasu) [inline bookie/tests/test_webviews/test_webviews.py]: Yes, the browser sends is_private: 'on'. The tests have been updated.

**Comment by sambuddhabasu** [comment]
:shipit:

**Pull request merged**